geojson.js

```diff
@@ -8,19 +10,23 @@
   // Converts an array of objects into a GeoJSON feature collection
   GeoJSON.parse = function(objects, params, callback) {
 
-    var geojson = {"type": "FeatureCollection", "features": []},
+    var geojson,
         settings = applyDefaults(params, this.defaults),
         propFunc;
 
     geomAttrs.length = 0; // Reset the list of geometry fields
     setGeom(settings);
     propFunc = getPropFunction(settings);
 
-    objects.forEach(function(item){
-      geojson.features.push(getFeature(item, settings, propFunc));
-    });
-
-    addOptionals(geojson, settings);
+    if (util.isArray(objects)) {
```

Comment:
Consider swapping this out for `Array.isArray`. `require`ing a module will break this library for vanilla JavaScript in the browser. Looking at the [source](https://github.com/nodejs/node/blob/master/lib/util.js#L834) of `util.isArray`, it's just a direct export of `Array.isArray`, so the code should be functionally the same.